Mastodon/Scene/HomeTimeline/HomeTimelineViewModel+LoadLatestState.swift

```diff
@@ -105,9 +105,10 @@ extension HomeTimelineViewModel.LoadLatestState {
 
         guard let viewModel else { return }
 
-        let latestFeedRecords = viewModel.dataController.records.prefix(APIService.onceRequestStatusMaxCount)
 
-        Task {
+        Task { @MainActor in
+            let latestFeedRecords = viewModel.dataController.records.prefix(APIService.onceRequestStatusMaxCount)
```

Comment:
I don't think this has something to do with the index-fix, but still it crossed my mind:

Do you know or remember the reason for `prefix(100)`? A few lines below we use the IDs of `latestFeedRecords` to determine new statuses: A status is new, if its ID isn't among those of max. 100 `latestFeedRecords` (from my understanding). But what if its the 101. or 102.?

I didn't find anything in the git history.

Reply:
Maybe the hard-coded value is related to the max result count of the (paginated) API? There should be a "Load missing posts" divider if there is a gap in between.

Reply:
Yes, I've seen that cell before, but it doesn't show up.

`limit` in the API-docs defaults to 20 and has a maximum value of 40. So using 100 as clientside default doesn't make a lot of sense to me.

I'll look into it :)